lib/ansible/plugins/callback/splunk.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8 -*-
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: splunk
+    type: aggregate
+    short_description: Sends task result events to Splunk HTTP Event Collector
+    author: "Stuart Hirst <support@convergingdata.com>"
+    description:
+      - This callback plugin will send task results as JSON formatted events to a Splunk HTTP collector.
+      - The companion Splunk Monitoring & Diagnostics App is available here: "https://splunkbase.splunk.com/app/4023/"
```

Comment:
The colon is causing the docs-build error.

Reply:
Thanks and updated